### ISSUE TITLE
PR Actions Are Not Shown When Title Is Long

### ISSUE THREAD

**Issue by hediet** [microsoft/vscode#227598]
When the PR title is very long, it is very difficult to open the multi file diff editor.

![Image](https://github.com/user-attachments/assets/5f6f4b19-df57-4a8a-be5c-b31efac7ac9c)

**Comment by alexr00** [comment]
I think you must have `workbench.list.horizontalScrolling` set. I can only repro if `workbench.list.horizontalScrolling` is set to true.

**Comment by hediet** [comment]
Yes, I forgot about that setting. Even if that setting is set, I think the title should have a max length.
